test/TestProjects/ExactMatchFlattenInheritance/Generated/AzureResourceFlattenModel1Container.cs

```diff
@@ -43,26 +43,27 @@ internal AzureResourceFlattenModel1Container(ResourceOperationsBase parent) : ba
 
         // Container level operations.
 
-        /// <summary> The operation to create or update a AzureResourceFlattenModel1. Please note some properties can be set only during creation. </summary>
         /// <param name="name"> The String to use. </param>
         /// <param name="parameters"> The AzureResourceFlattenModel1 to use. </param>
-        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="CancellationToken.None" />. </param>
-        public virtual Response<AzureResourceFlattenModel1> CreateOrUpdate(string name, AzureResourceFlattenModel1Data parameters, CancellationToken cancellationToken = default)
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="name"/> or <paramref name="parameters"/> is null. </exception>
+        public Response<AzureResourceFlattenModel1> Put(string name, AzureResourceFlattenModel1Data parameters, CancellationToken cancellationToken = default)
```

Comment:
If a resource named their create method incorrectly we should rename it to CreateOrUpdate.  We shouldn't rename all PUT methods but there is one that represents CreateOrUpdate behavior and in this case it appears to be that way

Reply:
I have changed this back